x-pack/plugins/cases/public/components/user_actions/index.test.tsx

```diff
@@ -452,18 +356,160 @@ describe(`UserActions`, () => {
       const props = {
         ...defaultProps,
         userProfiles: userProfilesMap,
-        caseUserActions: isolateAction,
         data: {
           ...defaultProps.data,
           comments: [hostIsolationComment({ createdBy: { profileUid: userProfiles[0].uid } })],
         },
       };
 
+      useFindCaseUserActionsMock.mockReturnValue({
+        ...defaultUseFindCaseUserActions,
+        data: { userActions: isolateAction },
+      });
+
       appMockRender.render(<UserActions {...props} />);
 
       expect(screen.getByTestId('case-user-profile-avatar-damaged_raccoon')).toBeInTheDocument();
       expect(screen.getByText('DR')).toBeInTheDocument();
       expect(screen.getByText('Damaged Raccoon')).toBeInTheDocument();
     });
   });
+
+  describe('pagination', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('Loading spinner when user actions loading and displays fullName/username', () => {
```

Comment:
nit: let's remove "and displays fullName/username" as we do not check the full name & username.